tests/pos/indent.scala

```diff
@@ -80,11 +84,15 @@ object Test:
 
 class Test2:
   self =>
-  def foo = 1
+  def foo(x: Int) =
+    if x < 0 then throw
```

Comment:
Do we really want to allow this? I somehow parse this as:

```scala
if x < 0 then { throw();
  val ex = ...
}
```

Reply:
For consistency I think it is better to allow it. The given expression abuses indentation rules for sure. It should be formatted like this:
```
if x < 0 then
   throw
       ...
```